starport/interface/cli/starport/cmd/network_chain_create.go

```diff
@@ -16,33 +16,48 @@ import (
 )
 
 const (
-	flagChainID = "chain"
-	flagSource  = "source"
-	flagRev     = "rev"
+	flagRev = "rev"
 )
 
 // NewNetworkChainCreate creates a new chain create command to create
 // a new network.
 func NewNetworkChainCreate() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "create",
+		Use:   "create [chain] [source]",
 		Short: "Create a new network",
 		RunE:  networkChainCreateHandler,
 	}
-	c.Flags().String(flagChainID, "", "Chain ID")
-	c.Flags().String(flagSource, "", "Git repository of the chain's source code (local or remote)")
 	c.Flags().String(flagRev, "", "Git revision of the source code")
 	return c
 }
 
 func networkChainCreateHandler(cmd *cobra.Command, args []string) error {
-	values, err := cliquiz.ValuesFromFlagsOrAsk(cmd.Flags(), "Please, provide:\n",
-		cliquiz.NewFlag(flagChainID, true),
-		cliquiz.NewFlag(flagSource, true),
-		cliquiz.NewFlag(flagRev, false),
+	// collect required values.
+	var (
+		chainID string
+		source  string
+		rev, _  = cmd.Flags().GetString(flagRev)
```

Comment:
Shouldn't we handle the error returned? don't know what would cause an error when getting flag though

Reply:
In our case, this should be fine because `err` seems like returned when trying to accessing to a not defined flag.